edb/tools/toy_eval_model.py

```diff
@@ -679,12 +679,12 @@ def eval_Group(node: qlast.GroupQuery, ctx: EvalContext) -> Result:
         Tuple[Tuple[ByElement, ...], Tuple[Data, ...]],
         Tuple[Dict[ByElement, Data], List[Data]]
     ] = {}
-    # Rebuild the set tuple from all_keys to both deduplicate
+    # Rebuild the set tuples from all_keys to both deduplicate
```

Comment:
Dedup comment now outdated?

Reply:
It still deduplicates internally to the key, which is actually what the comment *originally* referred to